spec/support/vcr_cassettes/search/success.yml

```diff
@@ -55,7 +55,11 @@ http_interactions:
       encoding: ASCII-8BIT
       string: '{
         "query": "benefits",
-        "web": {
+        "pagination": {
```

Comment:
We actually won't want this `pagination` object hard coded into the cassette, as we are appending it to this cassette's raw response in our `Search::Response` class.